crosswalks/publiccode.csv

```diff
@@ -0,0 +1,74 @@
+Property,publiccode
+codeRepository,url
+programmingLanguage,
+review,
+runtimePlatform,platforms
+targetProduct,platforms
```

Comment:
I was about to debate this, because (quoting [this old comment of mine](https://github.com/codemeta/codemeta/pull/300#issuecomment-1619960326)):

> In software engineering, one would say their app targets Windows, Flash, or the Web, to mean what product they aim to support, in the same way one targets an audience or democraphic.

However, given the description in https://docs.italia.it/italia/developers-italia/publiccodeyml-en/en/master/schema.core.html#key-platforms I think it might actually be relevant.